vsphere/image/files/etc/systemd/system/kata-agent.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Kata Agent
+After=network.target
+
+[Service]
+ExecStart=/usr/local/bin/kata-agent
```

Comment:
Something similar to https://github.com/confidential-containers/cloud-api-adaptor/pull/339 wil have to be done here, as in, passing the agent configuration and also creating the configuration file.

Reply:
Thanks for the review and the useful info. I updated kata-agent.service to reflect this change based on changes made in #339